docs/src/prototypes/dropdowns/MentionsWithDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import keyboardKey from 'keyboard-key'
+import { Dropdown, DropdownProps } from '@stardust-ui/react'
+
+import { atMentionItems } from './dataMocks'
+import { insertTextAtCursorPosition } from './utils'
+import { PortalAtCursorPosition } from './PortalAtCursorPosition'
+
+interface MentionsWithDropdownState {
+  dropdownOpen?: boolean
+  searchQuery?: string
+}
+
+const editorStyle: React.CSSProperties = {
+  backgroundColor: 'lightgrey',
+  padding: '5px',
+  minHeight: '100px',
+  outline: 0,
+}
+
+class MentionsWithDropdown extends React.Component<{}, MentionsWithDropdownState> {
+  private readonly initialState: MentionsWithDropdownState = {
+    dropdownOpen: false,
+    searchQuery: '',
+  }
+
+  private contendEditableRef = React.createRef<HTMLDivElement>()
+
+  state = this.initialState
+
+  render() {
+    const { dropdownOpen, searchQuery } = this.state
+
+    return (
+      <>
+        <div
+          contentEditable
+          ref={this.contendEditableRef}
+          onKeyUp={this.handleEditorKeyUp}
+          style={editorStyle}
+        />
+        <PortalAtCursorPosition open={dropdownOpen}>
+          <Dropdown
+            defaultOpen={true}
+            inline
+            search
+            items={atMentionItems}
+            toggleIndicator={null}
+            searchInput={{
+              input: { autoFocus: true, size: searchQuery.length + 1 },
```

Comment:
seems that `size` prop won't work, as there is no `size` prop declared for `DropdownSearchInput`. Am I missing something?

Reply:
`size` is an HTML attribute:

_unsigned long: Returns / Sets the element's size attribute, containing visual size of the control. This value is in pixels unless the value of type is text or password, in which case, it is an integer number of characters. Applies only when type is set to text, search, tel, url, email, or password; otherwise it is ignored.
(see: https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement)_

We need it to resize the input while typing; useful for mentioning people in the middle of text:
![screen recording 2019-02-22 at 14 48 06](https://user-images.githubusercontent.com/5442794/53246441-fc817f80-36b0-11e9-8eef-a519272b298d.gif)

Reply:
got it, thanks for explanation - actually, the fact that we are overlapping with DOM property's name seems a bit disturbing to me. But this is, definitely, a discussion aside from this PR.